quickjs.h

```diff
@@ -676,13 +676,20 @@ JS_BOOL JS_SetConstructorBit(JSContext *ctx, JSValueConst func_obj, JS_BOOL val)
 JSValue JS_NewArray(JSContext *ctx);
 int JS_IsArray(JSContext *ctx, JSValueConst val);
 
+typedef struct InlineCache InlineCache;
+
 JSValue JS_GetPropertyInternal(JSContext *ctx, JSValueConst obj,
                                JSAtom prop, JSValueConst receiver,
-                               JS_BOOL throw_ref_error);
+                               InlineCache *ic, JS_BOOL throw_ref_error);
+JSValue JS_GetPropertyInternalWithIC(JSContext *ctx, JSValueConst obj,
```

Comment:
Can we / should we avoid having this in the public API?

Reply:
This function and JS_SetPropertyInternalWithIC indeed don't have to be public.